Controller/ContentTypeController.php

```diff
@@ -350,4 +372,23 @@ public function deleteContentTypeAction(Request $request, $contentTypeId)
 
         return $this->redirectToRouteAfterFormPost('admin_contenttypeView', ['contentTypeId' => $contentTypeId]);
     }
+
+    /**
+     * Return the highest prioritized language that exists in $languageCodes, or if none, return $fallbackLanguageCode.
+     *
+     * @param string[] $languageCodes Language codes
+     * @param string $fallbackLanguageCode Fallback language code
+     *
+     * @return string Language code
+     */
+    private function getPrioritizedLanguage($languageCodes, $fallbackLanguageCode)
```

Comment:
The calls to `getPrioritzedLanguage` all read the arguments from the same `ContentTypeDraft`, unless I have missed something.

You could maybe simplify by making this method accept a ContentTypeDraft instead of language codes. It could then take care of the `array_keys` logic.